### PR TITLE
Grey out

### PR DESCRIPTION
**Pull request by Shamsco**
- Grey out Nodes on Group select
- Link Hover
- Zooming

### CODE REVIEW

**Review by mshinoy — APPROVED** [review]
I don't see any issues. Do check with @WafaWaheedaS also. Since I am not that well versed in d3

File: src/app/disjointed-graph/disjointed-graph.component.ts [hunk 1]
```diff
@@ -289,15 +351,21 @@ export class DisjointedGraphComponent implements OnInit {
...
-      .attr('class', classSuffix)
-      .style('border', '1px solid black');
+      .attr('class', classSuffix);
     return svg;
   }
 
-
   //Method to style the SVG
   private styleSVG(svg: any) {
     svg.append('style').text(`
```
Comment (WafaWaheedaS) [inline src/app/disjointed-graph/disjointed-graph.component.ts:360]: Let's move all styling to the respective component css file.

File: src/app/disjointed-graph/disjointed-graph.component.ts [hunk 2]
```diff
@@ -359,7 +437,7 @@ export class DisjointedGraphComponent implements OnInit {
       .attr('class', 'tooltip nodeTooltip')
       .style('opacity', 0)
       .attr('width', this.width / 4)
-      .attr('height', this.height / 4);
+      .attr('height', this.height / 2);
   }
   //Method to update the Graph simulation
   private graphUpdates(
```
Comment (WafaWaheedaS) [inline src/app/disjointed-graph/disjointed-graph.component.ts:443]: Can you please elaborate on what this function does?

File: src/app/disjointed-graph/disjointed-graph.component.html [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
\ No newline at end of file
+<div position="relative" id="Disjointed">
+    <div class="toolbar">
+        <mat-slide-toggle [(ngModel)]="gChecked" (click)="showGroups()">Groups</mat-slide-toggle>
+        <mat-slide-toggle [(ngModel)]="lChecked" (click)="showLinks()">Links</mat-slide-toggle>
+    </div>
+</div>
+
+<style>
+    div.toolbar {
```
Comment (WafaWaheedaS) [inline src/app/disjointed-graph/disjointed-graph.component.html]: Please move the styling to respective component css file

**Review by WafaWaheedaS — APPROVED** [review]

**Pull request merged**